src/components/DataTable/DataTable.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { DataTable, DataTableBody, DataTableEmpty, DataTableHeader } from "."
+
+export default {
+  title: "Components/Data Table",
+}
+
+const Template = {
+  children: "Data Table",
+}
+
+export const Empty = () => <DataTableEmpty />
```

Comment:
Can we wrap this in datatable component to better demo how it will look like in context its intended for?